packages/multi-trait-rubric/controller/src/index.js

```diff
@@ -51,3 +52,35 @@ export const createCorrectResponseSession = (question, env) => {
     }
   });
 };
+
+
+export const validate  = (model, config) => {
+  const { scales } = model;
+  const errors = {};
+  const traitsErrors = {};
+
+  (scales || []).forEach((scale, scaleIndex) => {
+    const { traits = [] } = scale;
+    const scaleErrors = {};
+    traits.forEach((trait, traitIndex) => {
+      if(!trait.name || trait.name === '<div></div>') {
+        scaleErrors[traitIndex] = 'Trait names should not be empty.';
+      }
+      else{
+        const identicalTraitName = traits.slice(traitIndex + 1).some(t => this.markupToText(t.name) === this.markupToText(trait.name));
```

Comment:
I don't think this. works here

Reply:
hmm, what do you? I tested and seems like the duplicate one is found

Reply:
markupToText is exported from another file so should probably be "markupToText" instead of "this.markupToText"